app/modules/sightings/resources.py

```diff
@@ -614,48 +613,14 @@ def delete(self, sighting):
                 code=400,
             )
 
-        response_data = None
-        if response.ok:
-            response_data = response.json()
-
-        if not response.ok or not response_data.get('success', False):
```

Comment:
These would never have been hit as this scenario results in a houston exception which is handled above